actors/builtin/reward/reward_actor.go

```diff
@@ -162,10 +163,10 @@ func (a Actor) UpdateNetworkKPI(rt runtime.Runtime, currRealizedPower *abi.Stora
 		// st.Epoch == rt.CurrEpoch()
 		for st.Epoch < rt.CurrEpoch() {
 			// Update to next epoch to process null rounds
-			st.updateToNextEpoch(*currRealizedPower)
```

Comment:
Is this branch possible to hit now that cron fires for every null round?  I do not think it is.

Reply:
Probably not, but I don't want to touch it here. Maybe in v2.